tools/dependencies/known-dependencies.txt

```diff
@@ -50,6 +50,7 @@ protostuff-runtime-1.8.0.jar
 scala-library-2.11.12.jar
 seatunnel-config-base-2.1.1.jar
 seatunnel-config-shade-2.1.1.jar
+seatunnel-jackson-2.3.1-SNAPSHOT-optional.jar
```

Comment:
we shouldn't add this line

Reply:
If we didn't add this，CI will not be passed. Do you have any another advice？

Reply:
i will take a look later:)